readthedocs/builds/tasks.py

```diff
@@ -206,3 +216,98 @@ def delete_inactive_external_versions(limit=200, days=30 * 3):
                 version.project.slug, version.slug,
             )
             version.delete()
+
+
+@app.task(
+    max_retries=1,
+    default_retry_delay=60,
+    queue='web'
+)
+def sync_versions_task(project_pk, tags_data, branches_data, **kwargs):
+    """
+    Sync the version data in the repo (on the build server).
+
+    Version data in the repo is synced with what we have in the database.
```

Comment:
```suggestion
    Sync the version data in the repo (from build server) into our database.

	Creates new Version objects for those tags/branches that we don't have tracked in our database and deletes Version objects for tags/branches that don't exists anymore in the repository.
```

I found the docstring a little ambiguous and I'm suggesting a change.